javascript/features/death_match/death_match_manager.js

```diff
@@ -101,6 +128,54 @@ export class DeathMatchManger {
             Math.round(statistics.accuracy * 100));
     }
 
+    // If this DM has teams we'll set the player in a team.
+    setPlayerTeam(player, zone, team = undefined) {
+        const location = DeathMatchLocation.getById(zone);
+        if (!location.hasTeams)
+            return;
+
+        if (!this.zoneTextDraws_.has(zone)) {
+            this.zoneTextDraws_.set(zone, -1);
+            wait(1).then(() => {
+                const textDraw = pawnInvoke('TextDrawCreate', 'ffs', 482, 311, 'Loading..');
```

Comment:
Please use the `components/text_draw/text_draw.js` interface rather than issuing Pawn commands. It'll be a bit tedious to convert this fully, but you're not doing anything outrageous.